sdk/src/errors.rs

```diff
@@ -321,3 +322,82 @@ pub(crate) fn status_to_error(status: tonic::Status) -> MomentoError {
         },
     }
 }
+
+enum LimitExceededMessageWrapper {
+    TopicSubscriptions,
+    OperationsRate,
+    ThroughputRate,
+    RequestSize,
+    ItemSize,
+    ElementSize,
+    Unknown,
+}
+
+impl LimitExceededMessageWrapper {
+    pub fn value(&self) -> String {
+        match self {
+            LimitExceededMessageWrapper::TopicSubscriptions => {
+                "Topic subscriptions limit exceeded for this account".to_string()
```

Comment:
Could also have this return `&static str` so then no need for `.to_string()` on the variants